stream_alert/athena_partition_refresh/main.py

```diff
@@ -139,14 +137,21 @@ def __init__(self, config, **kwargs):
             results_key_prefix (str): The S3 key prefix to store Athena results
         """
         self.config = config
+        self.prefix = self.config['global']['account']['prefix']
 
         region = self.config['global']['account']['region']
         self.athena_client = boto3.client('athena', region_name=region)
 
-        # Format the S3 bucket to store Athena query results
-        self.athena_results_bucket = 's3://aws-athena-query-results-{}-{}'.format(
-            self.config['global']['account']['aws_account_id'],
-            self.config['global']['account']['region'])
+        athena_config = self.config['lambda']['athena_partition_refresh_config']
+
+        # GEt the S3 bucket to store Athena query results
+        results_bucket = athena_config.get('results_bucket', '').strip()
+        if results_bucket == '':
+            self.athena_results_bucket = 's3://{}.streamalert.athena-results'.format(self.prefix)
+        elif results_bucket[:5] != 's3://':
+            self.athena_results_bucket = 's3://{}'.format(results_bucket)
+        else:
+            self.athena_results_bucket = results_bucket
```

Comment:
Instead of `if ... != ..., else` it's usually easier to read the positive case first (also you can use `startswith`):

```python
if not results_bucket:
    ...
elif results_bucket.startswith('s3://'):
    self.athena_results_bucket = results_bucket
else:
    self.athena_results_bucket = 's3://{}'.format(results_bucket)
```

Reply:
good suggestions!